src/view/ui/PropertyLine.svelte

```diff
@@ -34,7 +35,7 @@
     }
 </script>
 
-{#if !item.conditioned || (item.conditioned && `${property}`.length)}
+{#if !item.conditioned || (item.conditioned && `${property}` !== null)}
```

Comment:
\`${property}\` !== null

This will always be true

Reply:
That will still return true because `${null}` will create the string "null", which is not null

0 == null is true because `==` will coerce them both to false (as they're both falsey), one of the oddities of JS lol